ddp-workspace/projects/ddp-sdk/src/lib/services/logging.service.ts

```diff
@@ -67,9 +67,9 @@ export class LoggingService {
             body,
             { headers: new HttpHeaders({ 'Content-Type': 'application/json' }) }).pipe(
             catchError((error: any) => {
-                this.stackdriverErrorReporterService.handleError(`${this.LOG_SOURCE}: HTTP POST: ${url}`);
-                this.stackdriverErrorReporterService.handleError(error);
-                console.error.apply(window.console, error);
+                const message = `${this.LOG_SOURCE}: ${JSON.stringify(body)}`;
+                this.stackdriverErrorReporterService.handleError(message);
+                console.error.apply(window.console, message);
```

Comment:
`POST: https://us-central1-broad-ddp-prod.cloudfunctions.net/LoggingService` calls cloud function `LoggingService` to log a message to Stackdriver logging. But it can fail for an unknown reason. Fallback to `stackdriverErrorReporterService.handleError()` to send log same message to stackdriver logging.

Reply:
can this be tested to see if its working as planned ?